backend/core/test/user/user.e2e-spec.ts

```diff
@@ -570,4 +570,27 @@ describe("Applications", () => {
       true
     )
   })
+
+  it("should lower case email of new user", async () => {
+    const userCreateDto: UserCreateDto = {
+      password: "Abcdef1!",
+      passwordConfirmation: "Abcdef1!",
+      email: "TestingLowerCasing@LowerCasing.com",
+      emailConfirmation: "TestingLowerCasing@LowerCasing.com",
+      firstName: "First",
+      middleName: "Mid",
+      lastName: "Last",
+      dob: new Date(),
+    }
+    const mockWelcome = jest.spyOn(testEmailService, "welcome")
+    const res = await supertest(app.getHttpServer())
+      .post(`/user`)
+      .set("jurisdictionName", "Alameda")
+      .send(userCreateDto)
+    expect(mockWelcome.mock.calls.length).toBe(1)
+    expect(res.body).toHaveProperty("id")
+    expect(res.body).not.toHaveProperty("passwordHash")
+    expect(res.body).toHaveProperty("email")
+    expect(res.body.email).toBe("testinglowercasing@lowercasing.com")
```

Comment:
I would be nice to add a login check here, to see if login in with lower cased email works, but that requires you do use admin credentials to confirm a user (set confirmed_at through `PUT /user/:id` endpoint or by setting confirmed_at with UserRepository.